test/box-luatest/gh_10032_update_upsert_splice_test.lua

```diff
@@ -0,0 +1,78 @@
+local t = require('luatest')
+local server = require('luatest.server')
+-- Type data stored in tuple: string or varbinary.
+local types = {{type = 'str'}, {type = 'bin'}}
```

Comment:
Nit. Let's not mix imports and other variables.

Reply:
Good practice, fixed.